test/parallel/test-cli-eval.js

```diff
@@ -11,7 +11,6 @@ const child = require('child_process');
 const path = require('path');
 const nodejs = '"' + process.execPath + '"';
 
-
```

Comment:
Could you undo the whitespace changes in the lines your changes don’t touch? That helps when running e.g. `git blame` on a file